bokeh/models/formatters.py

```diff
@@ -278,6 +278,20 @@ class LogTickFormatter(TickFormatter):
     base to use. If unset, the formatter will use base 10 as a default.
     """)
 
+    use_superscript_exponents = Bool(False, help='''
+    Whether to ever display scientific values using exponent superscript.
+
+    Superscript fonts may not be rendered with the same size. Changing the major label
+    font may solve this issue.
```

Comment:
I'd suggest:
> Some fonts may render superscripts at different sizes. Changing the ...

Reply:
If there is a built-in font that seems to work well across browsers we could also consider making it the default

Reply:
In my specific case (Debian 9 / XFCE)  I had very good results with this font:
`p.axis.major_label_text_font = "sans"`

But I don't know if it is a good idea to making it default. In next days I will try on my colleagues computers (macOS and, I hope, Windows).